js/index.js

```diff
@@ -1,160 +1,58 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
+/*
  * @format
  * @flow
  */
 
-import {RNCGeolocation, GeolocationEventEmitter} from './nativeInterface';
-
-import invariant from 'invariant';
-import {logError, warning} from './utils';
-
-let subscriptions = [];
-let updatesEnabled = false;
-
-type GeoConfiguration = {
-  skipPermissionRequests: boolean,
-  authorizationLevel: 'always' | 'whenInUse' | 'auto',
-};
-
 type GeoOptions = {
   timeout?: number,
   maximumAge?: number,
   enableHighAccuracy?: boolean,
-  distanceFilter?: number,
-  useSignificantChanges?: boolean,
 };
 
-/**
- * The Geolocation API extends the web spec:
- * https://developer.mozilla.org/en-US/docs/Web/API/Geolocation
- *
- * See https://facebook.github.io/react-native/docs/geolocation.html
- */
 const Geolocation = {
-  /*
-   * Sets configuration options that will be used in all location requests.
-   *
-   * See https://facebook.github.io/react-native/docs/geolocation.html#setrnconfiguration
-   *
-   */
-  setRNConfiguration: function(config: GeoConfiguration) {
-    if (RNCGeolocation.setConfiguration) {
-      RNCGeolocation.setConfiguration(config);
-    }
+  setRNConfiguration: function() {
+    console.info('method not supported by the browser');
```

Comment:
I'm thinking that these should be something more like:

```javascript
throw new Error("method not supported by the browser");
```

The reason being that if the developer is calling them then they are likely making a mistake and the error should be as clear as possible. We shouldn't rely on them seeing a console log to understand why a method isn't doing what they expect.

Let me know if that makes sense.

Reply:
You're right, applied the same to the wrong argument passed as success callback.